overlord/snapstate/backend/link.go

```diff
@@ -319,6 +319,7 @@ func (b Backend) UnlinkSnap(info *snap.Info, linkCtx LinkContext, meter progress
 
 // ServicesEnableState returns the current enabled/disabled states of a snap's
 // services, primarily for committing before snap removal/disable/revert.
+// XXX: Not able to find where this is actually used?
```

Comment:
You are right, I see wrappers.QueryDisabledServices used in handlers.go via Backend.QueryDisabledServices but not this. It might make sense to keep this around until we are done with the services cleanups and drop it then if we haven't found a use for it by then.

Reply:
That makes sense! i'll leave it alone for now